frontend/packages/console-plugin-sdk/src/codegen/plugin-integration-tests.ts

```diff
@@ -0,0 +1,39 @@
+import * as _ from 'lodash';
+import {
+  PluginPackage,
+  filterActivePluginPackages,
+  PluginPackageFilter,
+  resolvePluginPackages,
+} from './plugin-resolver';
+
+const CONSOLE_PLUGIN_PREFIX = '@console/';
+const CONSOLE_INEGRATION_TESTS_PREFIX = '@console/integration-tests/';
+
+/**
+ * Return a map of integration tests.
+ */
+export const pluginIntegrationTests = (
+  monorepoRootDir: string = process.cwd(),
+  pluginFilter: PluginPackageFilter = filterActivePluginPackages,
+) => {
+  const shortName = (plugin: PluginPackage) => plugin.name.slice(CONSOLE_PLUGIN_PREFIX.length);
```

Comment:
We should check if `plugin.name` actually starts with `@console`, and only if so, strip it away.

I'd also favor `substring` over `slice` when working with strings.

```ts
const shortName = (plugin: PluginPackage) => plugin.name.startsWith(consolePkgScope)
  ? plugin.name.substring(consolePkgScope.length)
  : plugin.name;
```

Reply:
Nice ! doen